src/main/java/com/innobridge/ethpay/controller/PaymentController.java

```diff
@@ -58,4 +57,45 @@ public ResponseEntity<?> makePayment(
             return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(e.getMessage());
         }
     }
+
+    @PostMapping("/accept")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = OK, description = "Accept Transaction",
+                    content = @Content(mediaType = CONTENT_TYPE, schema = @Schema(implementation = Transaction.class)))
+    })
+    public ResponseEntity<?> acceptTransaction(@RequestParam String transactionId, @RequestParam String senderId) {
```

Comment:
You don't need to pass in the sender id

Reply:
receiver id you can get getAuthentication().getId()

  return ResponseEntity.ok(transactionService.acceptTransaction(transactionId, getAuthentication().getId()));